models/edr/run_results/seed_run_results.sql

```diff
@@ -0,0 +1,44 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+with dbt_run_results as (
+    select * from {{ ref('dbt_run_results') }}
+),
+
+dbt_seeds as (
+    select * from {{ ref('dbt_seeds') }}
+)
+
+SELECT
+    run_results.model_execution_id,
+    run_results.unique_id,
+    run_results.invocation_id,
+    run_results.query_id,
+    run_results.name,
+    run_results.generated_at,
+    run_results.status,
+    run_results.full_refresh,
+    run_results.message,
+    run_results.execution_time,
+    run_results.execute_started_at,
+    run_results.execute_completed_at,
+    run_results.compile_started_at,
+    run_results.compile_completed_at,
+    run_results.compiled_code,
+    run_results.adapter_response,
+    run_results.thread_id,
+    seeds.database_name,
+    seeds.schema_name,
+    coalesce(run_results.materialization, seeds.materialization) as materialization,
```

Comment:
looks like the `dbt_seeds` model does not have a `materialization` column, you can just do 
```suggestion
    run_results.materialization,
```

Reply:
Thank you, all test passed locally, I will check why

Reply:
Added tests to validate elementary view models including `seed_run_results`.